content/applications/finance/accounting/fiscal_localizations/localizations/peru.rst

```diff
@@ -195,6 +195,9 @@ in your configuration. In this case you need to consider:
   
 - Provide you SOL credentials.
 
+.. important::
+   When using direct connection with the SUNAT, the Sol User must be set with the Company RUT + User
```

Comment:
shouldn't it be `SOL` user ? (Also, I don't see any explanation on what the SOL means on the page :thinking:, if it is an acronym for something, it could be interesting to specify its representation on the first use of the acronym).

Reply:
You're right.

From what I understand, SOL is a bit like the Belgian Intervat. I guess that this is why it isn't explained here. So the description on line 94 explained its use well. I added the acronym detail in the same fashion as the other lines before. (I think that the `:abbr:` directive wouldn't work between `**` anyway...)

Thanks @Feyensv :)